### DISCUSSION TITLE
Add a viewer user role that can add annotations.

### DISCUSSION THREAD

**Discussion by westonforbes** [grafana/grafana#55692] [General]
It would be reeeeally helpful to be able to have a user role that can add/edit annotations without also giving them privileges to edit dashboards. Something like:
* Admin
* Editor
* **Annotator**
* Viewer
This would be incredibly useful to give to our quality technicians, so that when anomalous data shows up, they can investigate then make a note of findings.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
